pkg/chartverifier/checks/charttesting.go

```diff
@@ -139,10 +139,10 @@ func testRelease(
 	cleanupHelmTests bool,
 ) error {
 	if err := kubectl.WaitForDeployments(namespace, releaseSelector); err != nil {
-		return err
+		return fmt.Errorf("waiting for deployments: %w", err)
 	}
 	if err := helm.Test(namespace, release); err != nil {
-		return err
+		return fmt.Errorf("error testing release: %w", err)
```

Comment:
is these changes really needed for the script we submit in this PR to work?

Reply:
This change has been reverted.